.github/workflows/release.yml

```diff
@@ -12,12 +12,120 @@ on:
         required: true
         default: true
         type: boolean
+      release_version:
+        description: "Fill it if backport or RC. Otherwise with-defaults will be used"
+        required: false
+        type: string
 
 jobs:
-  setup:
+  build:
     runs-on: ubuntu-latest
     env:
+      SBT_RELEASE_NEXT_VERSION: ${{ (inputs.release_version && format('{0} {1} with-defaults', 'release-version', inputs.release_version)) || 'with-defaults' }}
       NU_DOCKER_UPDATE_LATEST: ${{ inputs.backport_release == false && inputs.release_candidate == false }}
+      JVM_OPTS: "-Xss6M -Xms4G -Xmx4G -XX:ReservedCodeCacheSize=256M -XX:MaxMetaspaceSize=2500M -DdockerUpLatest=false"
```

Comment:
1. Can we have this variable only on relese a.k. `name: "Build"` job? 
2. why `-DdockerUpLatest` is hardcoded?  shouldn't be passed `NU_DOCKER_UPDATE_LATEST` there?

Reply:
good catch